### PR TITLE
BanishedUser also destroy organization

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] feature?

## Description
BanishedUser also destroy organization

## Related Tickets & Documents
resolves https://github.com/forem/internalCommunity/issues/379

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

**Review by lightalloy — APPROVED** [review]
Looks good though buildkite complains :eyes:

File: spec/workers/moderator/banish_user_worker_spec.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 RSpec.describe Moderator::BanishUserWorker, type: :worker do
   include_examples "#enqueues_on_correct_queue", "high_priority", 1
 
+  # TODO: Remove this test because BanihedUser already covers it
```
Comment (Ridhwana) [inline spec/workers/moderator/banish_user_worker_spec.rb:6]:
```suggestion
  # TODO: Remove this test because BanishedUser already covers it
```

**Review by Ridhwana — APPROVED** [review]
Tried to re-run BuildKite 🤞🏼

File: spec/workers/moderator/banish_user_worker_spec.rb (hunk 1 above)
Comment (maestromac) [reply spec/workers/moderator/banish_user_worker_spec.rb:6]: I will follow up with this real soon

**Pull request merged**